src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowHotfixFinishMojo.java

```diff
@@ -177,9 +177,23 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             gitMergeNoff(hotfixBranchName);
 
             final String currentVersion = getCurrentProjectVersion();
+            if (useSnapshotInHotfix && ArtifactUtils.isSnapshot(currentVersion)) {
```

Comment:
Will it work with support branches and release branch?

Reply:
hotfix will merge into the release branch if the release branch exists. The hotfix of support branch is also a thing. :)

Will it work with them?